voxel51/users/api.py

```diff
@@ -844,6 +868,75 @@ def kill_job(self, job_id):
         res = self._requests.put(endpoint, headers=self._header)
         _validate_response(res)
 
+    def batch_start_jobs(self, job_ids):
+        '''Starts all jobs with the given IDs.
+
+        Args:
+            job_ids (list): the job IDs
+
+        Raises:
+            :class:`APIError` if the request was unsuccessful
+        '''
+        return self._batch_request("jobs", "start", job_ids)
+
+    def batch_archive_jobs(self, job_ids):
+        '''Archives all jobs with the given IDs.
+
+        Args:
+            job_ids (list): the job IDs
+
+        Raises:
+            :class:`APIError` if the request was unsuccessful
+        '''
+        return self._batch_request("jobs", "archive", job_ids)
+
+    def batch_unarchive_jobs(self, job_ids):
+        '''Unarchives all jobs with the given IDs.
+
+        Args:
+            job_ids (list): the job IDs
+
+        Raises:
+            :class:`APIError` if the request was unsuccessful
+        '''
+        return self._batch_request("jobs", "unarchive", job_ids)
+
+    def batch_update_jobs_ttl(self, job_ids, days):
```

Comment:
this will need to support `days=None, expiration_date=None` as per https://github.com/voxel51/api-py/pull/64